server/src/main/java/org/elasticsearch/action/admin/cluster/storedscripts/GetStoredScriptResponse.java

```diff
@@ -48,12 +74,24 @@ public StoredScriptSource getSource() {
     }
 
     @Override
-    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
-        source.toXContent(builder, params);
+    public boolean isFragment() {
+        return true;
+    }
```

Comment:
this method is not meant to be overridden. Please use ToXContentFragment instead of ToXContentObject if needed. Yet normally responses are objects rather than fragments, you sure about this?